woocommerce/woocommerce-ios
# Disable ghost cells selection

Closes https://github.com/woocommerce/woocommerce-ios/issues/2513.
Depends on https://github.com/wordpress-mobile/WordPressUI-iOS/pull/108.

## Description

This PR disables selection of tableview ghost content.

## Test

1. Update pods to install beta version of `WordPressUI`.
1. Open Reviews list (or any).
2. While loading tap on any ghost cells, confirm that they are **not** selectable.
3. After loading is finished, confirm that real reviews cells are selectable.

## Screenshots

before|after
--|--
![Simulator Screen Shot1](https://user-images.githubusercontent.com/3132438/151849814-1d7fc1d5-e297-4e01-b9cf-09f572b30742.png)|![Simulator Screen Shot2](https://user-images.githubusercontent.com/3132438/151849818-f3efc5d6-cdc6-4602-a3db-4b9bc13b0cf7.png)

---
